main.cpp

```diff
@@ -73,12 +25,24 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+void ToString() {
```

Comment:
ToString is supposed to return a String that can be printed... not output it to the console. That way it can be further processed to print, pdf, etc.

Reply:
So do we return a single string which is of the pattern "code major minor" am I right?

Reply:
Yes and if we print that string, it should come similar to the couts you have done below.

Reply:
Okay, I have made the require changes.